src/server/img_storage.rs

```diff
@@ -190,9 +190,13 @@ impl StoreImage {
     pub fn meta(&self, image: &Bytes, fmt: ImageFormat) -> HandlerResult<ImageMetrics> {
         let mut reader = ImageReader::new(Cursor::new(image));
         reader.set_format(fmt);
-        let img = reader
-            .decode()
-            .map_err(|_| HandlerErrorKind::BadImage("Image unreadable"))?;
+        let img = reader.decode().map_err(|e| {
+            let mut tags = Tags::default();
+            tags.add_extra("error", &e.to_string());
```

Comment:
I think both `url` and `format` are really worth adding as tags here too -- the easiest way to do that is to probably shuffle this method around a bit to have the same signature as `validate`.

Reply:
Sigh. 
the change was included in the merge https://github.com/mozilla-services/contile/pull/349/commits/e767a61d02cb42123647db4604497a8e68c68ca3